yarn-project/acir-simulator/src/avm/journal/journal.ts

```diff
@@ -150,9 +144,10 @@ export class AvmJournal {
    */
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 141 to 141]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Please fix comment.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/AztecProtocol/aztec-packages/4197?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->